### PR TITLE
feat: Initialize base project.

### PR DESCRIPTION
**Pull request by ExxiDauS**

### CODE REVIEW

**Review by pubethSri — COMMENTED** [review]
**Cool.** to add, I think in the constructor of flowchart we can create *Start* and add it at the first index of *flowchart* order (ArrayList) and change the attribute name to match naming rule *Order* -> *order*

**Review by KuhnphOn — CHANGES_REQUESTED** [review]
- Change getCode Method to convertToCode for less confusion.
- Encapsulate attribute in Flowchart.java

**Comment by ExxiDauS** [comment]
> **Cool.** to add, I think in the constructor of flowchart we can create _Start_ and add it at the first index of _flowchart_ order (ArrayList) and change the attribute name to match naming rule _Order_ -> _order_

Add the _Start_ . Does that mean I'll have to create a var?

**Comment by ExxiDauS** [comment]
- [ x ] Initialize start at Flowchart constructor.
- [ x ] Change the _Order_ attribute to correct the naming rule.
- [ x ] Change the name of the method to less complicated.
- [ x ] Encapsulate **Flowchart** fields.

File: Main_project/src/Flowchart.java
Comment (KuhnphOn) [inline Main_project/src/Flowchart.java:1]: Nice1

**Review by KuhnphOn — COMMENTED** [review]
I'm good with these changes. //Leave the comment here for others to review changes.

**Review by pubethSri — APPROVED** [review]
### **Very Cool.** Pretty sure it's ok to merge now.
sonic-boom-knuckles

**Review by KuhnphOn — APPROVED** [review]

**Pull request merged**